subsys/debug/cpu_load/cpu_load.c

```diff
@@ -216,7 +216,7 @@ void cpu_load_reset(void)
 static u32_t sleep_ticks_to_us(u32_t ticks)
 {
 	return IS_ENABLED(CONFIG_CPU_LOAD_ALIGNED_CLOCKS) ?
-	   (u32_t)(((u64_t)ticks * 1000000) / CONFIG_SYS_CLOCK_TICKS_PER_SEC) :
+	   (u32_t)(((u64_t)ticks * 1000000) / sys_clock_hw_cycles_per_sec()) :
```

Comment:
Tick means a specific thing is OS. Is the name of the argument (and function) correct? It seems that cycles are passed not ticks.
If this is true, please correct this misleading name.

Reply:
`ticks` comes from `TIMER` peripheral. At least to me, term `ticks` is not reserved from OS only.

Reply:
The fact that this name is very misleading is proven in the fixed code itself. Since the function is `sleep_ticks_to_us` somebody used `CONFIG_SYS_CLOCK_TICKS_PER_SEC` thinking that "tick" means system tick.

I agree that term "tick" can be used for many things in real life. But in OS it usually means time interval used for time bookkeeping. If it is used for any other thing it can lead to misunderstanding and bugs :(
https://docs.zephyrproject.org/latest/reference/kernel/timing/clocks.html
https://www.freertos.org/implementation/a00011.html
https://mynewt.apache.org/latest/os/modules/hal/hal_os_tick/hal_os_tick.html
https://www.kernel.org/doc/html/latest/timers/no_hz.html

```
... scheduling-clock interrupts (also known as “scheduling-clock ticks” or simply “ticks”)
```